pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/channel/ServiceUnitStateChannelImpl.java

```diff
@@ -1323,12 +1332,19 @@ private void handleBrokerDeletionEvent(String broker) {
         }
     }
 
+    private boolean channelDisabled() {
+        final var channelState = this.channelState;
+        if (channelState == Disabled || channelState == Closed) {
+            log.warn("[{}] Skip scheduleCleanup because the state is {} now", brokerId, channelState);
```

Comment:
We may need to make this log generic or remove it as it is used in multiple places.

Reply:
It's right, I'll change it in the new PR